drivers/gpio/gpio_utils.h

```diff
@@ -8,6 +8,15 @@
  * @file Header where utility code can be found for GPIO drivers
  */
 
+#include <stdbool.h>
+#include <stdint.h>
+#include <errno.h>
+
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/sys/slist.h>
+
 #ifndef ZEPHYR_DRIVERS_GPIO_GPIO_UTILS_H_
 #define ZEPHYR_DRIVERS_GPIO_GPIO_UTILS_H_
```

Comment:
Any particular reason for including the headers before this guard?

Reply:
argh, fixed thanks